### PR TITLE
feat(provider/ecs): Added an ECS Cluster reader

### PR DESCRIPTION
**Pull request by BrunoCarrier**
feat(provider/ecs): Added an ECS Cluster reader

@anotherchrisberry @jrsquared @christopherthielen @dkirillov

### CODE REVIEW

File: app/scripts/modules/ecs/ecsCluster/EcsCluster.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface EcsClusterDescriptor {
+  account: string;
+  region: string;
+  name: string;
+}
+
```
Comment (anotherchrisberry) [inline app/scripts/modules/ecs/ecsCluster/EcsCluster.ts]: whitespace, prefix with `I`

File: app/scripts/modules/ecs/ecsCluster/ecsCluster.read.service.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { module } from 'angular';
+
+import { API_SERVICE, Api } from 'core/api/api.service';
+import { EcsClusterDescriptor } from './EcsCluster';
+
+export class EscClusterReader {
+public constructor(private API: Api) { 'ngInject'; }
```
Comment (anotherchrisberry) [inline app/scripts/modules/ecs/ecsCluster/ecsCluster.read.service.ts]: indent this line

File: app/scripts/modules/ecs/ecsCluster/EcsCluster.ts (hunk 1 above)
Comment (BrunoCarrier) [reply app/scripts/modules/ecs/ecsCluster/EcsCluster.ts]: Changed as requested

File: app/scripts/modules/ecs/ecsCluster/ecsCluster.read.service.ts (hunk 2 above)
Comment (BrunoCarrier) [reply app/scripts/modules/ecs/ecsCluster/ecsCluster.read.service.ts]: Changed as requested

**Pull request merged**